src/inference/src/dev/performance_heuristics.cpp

```diff
@@ -8,7 +8,8 @@ namespace ov {
 
 MemBandwidthPressure mem_bandwidth_pressure_tolerance(const std::shared_ptr<ov::Model> model,
                                                       const float cache_size,
-                                                      const float memThresholdAssumeLimited) {
+                                                      const float memThresholdAssumeLimited,
+                                                      const ov::element::Type targetType) {
```

Comment:
```suggestion
                                                      const float mem_threshold_assume_limited,
                                                      const ov::element::Type& target_type) {
```
Detail: Is preferred for openvino lib.